backend/src/apiserver/server/pipeline_server.go

```diff
@@ -113,17 +113,14 @@ func (s *PipelineServer) createPipeline(ctx context.Context, pipeline *model.Pip
 	}
 	pipeline.Namespace = s.resourceManager.ReplaceNamespace(pipeline.Namespace)
 	// Check authorization
-	if common.IsMultiUserMode() {
-		resourceAttributes := &authorizationv1.ResourceAttributes{
-			Namespace: pipeline.Namespace,
-			Verb:      common.RbacResourceVerbCreate,
-			Group:     common.RbacPipelinesGroup,
-			Resource:  common.RbacResourceTypePipelines,
```

Comment:
ditto on missing `Resource` in the new code.

Reply:
See [here](https://github.com/kubeflow/pipelines/pull/9373/files#r1191530425)